spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/handler/RoutePredicateHandlerMapping.java

```diff
@@ -78,6 +78,7 @@ private static Integer getPortProperty(Environment environment, String prefix) {
 	protected Mono<?> getHandlerInternal(ServerWebExchange exchange) {
 		// don't handle requests on management port if set and different than server port
 		if (this.managementPortType == DIFFERENT && this.managementPort != null
+				&& exchange.getRequest().getLocalAddress() != null
```

Comment:
Makes sense, you could use `ObjectUtils.isEmpty` here as well

Reply:
`exchange.getRequest().getLocalAddress()`  returns `InetSocketAddress` 
I can change to Objects.nonNull() if you want

Reply:
No big deal in this case, it might look a bit cleaner, but functionally its the same.  More of an FYI